src/util/outputNameTemplate.js

```diff
@@ -65,15 +65,23 @@ export function getOutSegError({ fileNames, filePath, outputDir, safeOutputFileN
 // This is used as a fallback and so it has to always generate unique file names
 // eslint-disable-next-line no-template-curly-in-string
 export const defaultOutSegTemplate = '${FILENAME}-${CUT_FROM}-${CUT_TO}${SEG_SUFFIX}${EXT}';
-
+let currentTimestamp = Date.now();
```

Comment:
I think this timestamp should be updated every time, not just once when the app starts?

Reply:
I'm not 100% sure how the state management works for this, but after pulling your changes everything seems to work like how I expect (checked here: https://i.imgur.com/bnX8Ds4.mp4), so to me it's all good. I noticed that the timstamp still resets whenever the output filename's input box gets changed, which is good enough for me.